workspaces/adventure-pack/goodies/kotlin/src/common/TreeNode.kt

```diff
@@ -0,0 +1,3 @@
+package common
+
+public data class TreeNode(val value: Int, var left: TreeNode? = null, var right: TreeNode? = null)
```

Comment:
Nice, is this what LeetCode's TreeNode implementation is? I'd like to stay consistent with that, since that's what we'll have at runtime.